framework/src/main/java/org/checkerframework/common/value/statically-executable.astub

```diff
@@ -334,6 +334,57 @@ class String{
     @StaticallyExecutable static String valueOf(Object obj) ;
 }
 
+package java.math;
+
+class BigInteger {
+    @StaticallyExecutable BigInteger(@IntRange(from = -1, to = 1) int signum, byte[] magnitude);
+    @StaticallyExecutable BigInteger(byte[] val);
+    @StaticallyExecutable BigInteger(int bitLength, int certainty, Random rnd);
+    @StaticallyExecutable BigInteger(int numBits, Random rnd);
+    @StaticallyExecutable BigInteger(String val);
+    @StaticallyExecutable BigInteger(String val, @IntRange(from = 2, to = 36) int radix);
```

Comment:
None of these constructors are `@StaticallyExecutable`, because they are not guaranteed to return the exact same value given the same arguments. After all, constructors usually allocate new objects! The other annotations here seem fine, but those should all be removed. As a general rule, I'd suggest only writing annotations you're really confident of in the JDK - they will be trusted by the compiler!